test/onnx/test_fx_op_consistency.py

```diff
@@ -561,6 +562,19 @@
         matcher=lambda sample: not isinstance(sample.kwargs.get("weight"), int),
         reason="ONNX SoftmaxCrossEntropyLoss op only accept argument[weight] is int type",
     ),
+    xfail(
+        "nn.functional.embedding_bag",
+        matcher=lambda sample: sample.kwargs.get("max_norm") is not None,
+        reason="Torchlib does not support aten::embedding_renorm, emitted when 'max_norm' is not None",
+    ),
+    xfail(
+        "nn.functional.embedding_bag",
+        matcher=lambda sample: sample.kwargs.get("padding_idx") is not None or True,
```

Comment:
@justinchuby unfortunately `padding_idx=-1` is emitted, so we end up xfailing every subtest.

I think when adding op in torchlib we need to enable test on pytorch export side as well to reduce friction, thoughts?